analysed-packages/libestr/version-0.1.11/libestr-0.1.11-SPDX2TV.spdx

```diff
@@ -0,0 +1,2048 @@
+LicenseInfoInFile: LicenseRef-GPL-3.0-or-later
+LicenseInfoInFile: LicenseRef-Libtool-exception
+FileCopyrightText: <text> Copyright (C) 1996-2015 Free Software Foundation, Inc. 
+Copyright (C) 2004-2015 Free Software Foundation, Inc.
+Copyright (C) 2010-2015 Free Software Foundation, Inc.  </text>
+
+##File
+
+FileName: libestr-0.1.11.tar.gz/libestr-0.1.11.tar/libestr-0.1.11/depcomp
+SPDXID: SPDXRef-item440268
+FileChecksum: SHA1: 217e04a768f3c9407fcc1cca1838651e6aa02326
+FileChecksum: SHA256: 534cc0f4c4bfc8f2c5a674741e46abfcc033776a011c3d5bc48cad7d0394ef91
+FileChecksum: MD5: 636e2d2075c5ec88671ae84f3e69d6dd
+LicenseConcluded: LicenseRef-fossology-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-fossology-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic
+FileCopyrightText: <text> Copyright (C) 1999-2017 Free Software Foundation, Inc. </text>
+
+##File
+
+FileName: libestr-0.1.11.tar.gz/libestr-0.1.11.tar/libestr-0.1.11/compile
+SPDXID: SPDXRef-item440248
+FileChecksum: SHA1: 03a05e37267c2f1c9fea4f2a6a15f8a7beb91c0d
+FileChecksum: SHA256: 41856428226c049fa0ea4ca7a574c28416de2769b55ed7feea4ab5e60283af16
+FileChecksum: MD5: 500b9244caa7a7ab23ece1db37efa76d
+LicenseConcluded: LicenseRef-fossology-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-fossology-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic-3.0
+FileCopyrightText: <text> Copyright (C) 1999-2014 Free Software Foundation, Inc. Written by Tom Tromey <tromey@cygnus.com>.
+Copyright (C) 1999-2014 Free Software Foundation, Inc. Written by Tom Tromey <tromey@cygnus.com>. </text>
+
+##File
+
+FileName: libestr-0.1.11.tar.gz/libestr-0.1.11.tar/libestr-0.1.11/missing
+SPDXID: SPDXRef-item440236
+FileChecksum: SHA1: fced81554799feb0f10fb56e3a06acb92c7ce7b0
+FileChecksum: SHA256: b1a337b731bbb58846d31ca2f5143c293a455fa41b481e236e89a9016d96b034
+FileChecksum: MD5: 5f2a276ee7b8decec541391229d03591
+LicenseConcluded: LicenseRef-fossology-GPL-2.0-or-later-WITH-Autoconf-exception-generic
+LicenseInfoInFile: LicenseRef-GPL-2.0-or-later
+LicenseInfoInFile: LicenseRef-fossology-Autoconf-exception
+LicenseInfoInFile: LicenseRef-Autoconf-exception-generic
+FileCopyrightText: <text> Copyright (C) 1996-2014 Free Software Foundation, Inc. Originally written by Fran,cois Pinard <pinard@iro.umontreal.ca>, 1996.
+Copyright (C) 1996-2014 Free Software Foundation, Inc. Originally written by Fran,cois Pinard <pinard@iro.umontreal.ca>, 1996. </text>
+
+##File
+
+FileName: libestr-0.1.11.tar.gz/libestr-0.1.11.tar/libestr-0.1.11/ChangeLog
+SPDXID: SPDXRef-item440244
+FileChecksum: SHA1: d3aee29bb90b9b30968a4e3bfa7c984162bf18a8
+FileChecksum: SHA256: ab65a2aa88a708c8fde806fab2d21b9341a00700e5f500aae48474ac807b9f33
+FileChecksum: MD5: de89a6b8b900ada1ba2a49d76d07ceec
+LicenseConcluded: LicenseRef-fossology-LGPL
```

Comment:
Please remove the concluded license. The ChangLog is not explicitly licensed, furthermore this is not a valid license, in case there would be a phrase like "this file is licensed under LGPL" the correct conclusion would be LGPL-2.0-or-later.
The correct concluded license here is "NOASSERTION"
Please remove also the content of the license comment